uftrace.c

```diff
@@ -1324,10 +1324,6 @@ int main(int argc, char *argv[])
 	if (opts.use_pager && opts.mode != UFTRACE_MODE_LIVE)
 		start_pager(pager);
 
-	/* the srcline info is used for TUI status line by default */
-	if (opts.mode == UFTRACE_MODE_TUI)
-		opts.srcline = true;
-
```

Comment:
please check this change.

Reply:
This is needed because TUI always has to show source location info at the footer if available. We might have to change the type of `srcline` from `bool` to `int` and use this case as `1` and other explicit `srcline` option goes to `2`. It would be better to be some enum type that represents useful meaning.

Reply:
Do you mean change type of srcline in "struct opts" or apply it in tui?